DOCS.md

```diff
@@ -0,0 +1,61 @@
+# Documentation best practices
+This doc is still work in progress and will constantly be updated.
+
+1. Keep the examples in the stories minimal, without using unnecessary props that don't serve the purpose of demonstrating the story unless the Vibe's designers specifically request it.
+2. Whenever possible, avoid adding titles to the variants displayed within the story to maintain clear and simpler story code.
+3. When laying out components, utilize Vibe's Flex component instead of implementing your own flex layout using CSS. Refer to the code example below for clarification.
+```js
+<Flex direction={Flex.directions.COLUMN} gap={Flex.gaps.MEDIUM}>
```

Comment:
explain that using the Flex layout is needed only in case more than a single example is shown in a story block (oppose to simple examples)